public/scripts/tagsList.js

```diff
@@ -21,7 +21,7 @@ function createTagObject(content) {
     let template = document.createElement('template')
     template.innerHTML =
         `<div class="tag-element">
-            <span class="tag-element-text" data-value=${content}>${content}</span> 
+            <span class="tag-element-text" data-value="${content}">${content}</span> 
```

Comment:
Careful: if `content` has double-quote char, it'll break your markup. Consider escape it.

Reply:
Or you can use single quotes.